### DISCUSSION TITLE
Nova::userTimezone dos not reflect to DateTime::make field

### DISCUSSION THREAD

**Discussion by beshoo** [laravel/nova-issues#5644] [Ideas & Feature Requests]
- Laravel Version: 10.x
- Nova Version: 4.x Latest
- PHP Version: 8.x
- Database Driver & Version: Mysql 8.x
- Operating System and Version: Linux
- Browser type and version: Chrome
---------------------------------------
- **Larvel timezone UTC**
- Mysql timezone UTC
- PC as localhost timezone UTC

### Description:
I want to show the Time reflecting the selected time zone from User..

NovaServiceProvider
     */
```
    public function boot()
    {
        parent::boot();
        $this->getNovaMenu();
        Nova::userTimezone(function (Request $request) {
            return $request->user()?->time_zone; // There is a filed in database 'time_zone'
        });
    }
```

Resource View 
```
            DateTime::make('Presence', 'presence')->displayUsing(function ($value) use ($request) {
               return $value?->format('l / h:i');
            })->sortable(),
```

Display dos not respect the timezone user selects , to solve this, for now, am using :

```
            DateTime::make('Presence', 'presence')->displayUsing(function ($value) use ($request) {
                if($value)
                {
                    return sprintf('%s', Carbon::parse($value)->setTimezone($request->user()->time_zone ?: 'UTC')->format('l / h:i'));
                }
                return null;
            })->sortable(),
```

**Comment by crynobone** [comment]
This is expected behavior for Laravel Nova 4.

**Comment by beshoo** [comment]
Then what is the benefit of "Nova::userTimezone"! if I will change each DatTime timezone by hand!

**Reply by crynobone** [reply]
You are only required if you use `displayUsing()`, The current behavior is that every value is output using application timezone using ISO 8601 format and converted to user timezone on the front end.

Changing this on patch release can also cause breaking changes to components depending on this setup.
